src/Command/Environment/EnvironmentSshCommand.php

```diff
@@ -20,7 +20,7 @@ protected function configure()
             ->setName('environment:ssh')
             ->setAliases(['ssh'])
             ->addArgument('cmd', InputArgument::OPTIONAL, 'A command to run on the environment.')
-            ->addOption('pipe', null, InputOption::VALUE_NONE, "Output the SSH URL only.")
+            ->addOption('pipe', null, InputOption::VALUE_NONE, 'Output the SSH URL only.')
```

Comment:
Yeah, just because there's no need for double quotes here, sorry for diff annoyance 🤣